fsharp-backend/tests/testfiles/httpclient.tests

```diff
@@ -180,6 +180,36 @@ HttpClient.get_v3 "file://localhost//etc/passwd" {} {} = Test.typeError_v0 "Inte
 HttpClient.get_v4 "file://localhost//etc/passwd" {} {} = Test.typeError_v0 "Internal HTTP-stack exception: Unsupported protocol"
 HttpClient.get_v5 "file://localhost//etc/passwd" {} {} = Error "Unsupported protocol"
 
+[fn.getContentType response:any]
+Dict.get_v2 response.headers "content-type"
+
+// A request to https://google.com results in a 301 to https://www.google.com
+// That 301 includes a response header of
+// `content-type`: `"text/html; charset=ISO-8859-1"`
+//
+// The subsequent response to https://www.google.com includes a response header
+// of `content-type`: `"text/html; charset=utf-8"`.
+//
+// The expected/correct content-type header that a Dark user should receive is
+// `"text/html; charset=utf-8"`, since that's the charset of the actual content
```

Comment:
```suggestion
// A request to https://google.com results in a 301 to https://www.google.com
// That 301 includes a response header of
// `content-type`: `"text/html; charset=utf-8"`
//
// The subsequent response to https://www.google.com includes a response header
// of `content-type`: .`"text/html; charset=ISO-8859-1"`
//
// The expected/correct content-type header that a Dark user should receive is
// `"text/html; charset=ISO-8859-1"`, since that's the charset of the actual content
```

Reply:
Is that `.` supposed to be there?